content/docs/user/artifact_gen/index.md

```diff
@@ -160,24 +160,23 @@ We only support sbt 1+, not sbt 0.13.
 {{< /alert >}}
 
 {{< alert warning >}}
-If you use the 'it' config, you have to use a custom build command as the defauit one is for the 'test' config: `sbt -J-Xss100M ;clean;it:assembly -batch --error`
+If you use the 'it' config, you have to use a custom build command as the default one is for the 'test' config: `sbt ;clean;it:assembly`
```

Comment:
Restore Xss. We really need this because users might generate scenarios with super long method chaining.